various improvements

sonnyp opened the pull request:
Commits should be self explanatory.

I couldn't use GNOME Builder for some reason (unrelated to these changes but didn't look into it) and I wanted to make sure the sandbox was enabled so I did

```sh
cd gnomit
flatpak-builder --user --force-clean --repo=repo --install-deps-from=flathub flatpak ind.ie.Gnomit.json
flatpak --user remote-add --no-gpg-verify --if-not-exists Gnomit repo
flatpak --user install --reinstall --assumeyes Gnomit ind.ie.Gnomit
```

sonnyp committed:
hide the desktop entry

sonnyp committed:
use relative builddir path in flatpak manifest

sonnyp committed:
update to gspell 1.8.3

sonnyp committed:
update runtime version to 3.36

sonnyp committed:
ignore flatpak artefact

sonnyp committed:
remove network permission

sonnyp committed:
add Sonny Piers to contributors

sonnyp committed:
remove flatpak dconf permissions

sonnyp committed:
reduce size of screenshot in README

sonnyp committed:
fix gjs warning by using ByteArray.toString

TingPing commented on data/ind.ie.Gnomit.desktop.in:
```diff
@@ -6,3 +6,4 @@ Type=Application
 Categories=Development;Utility;RevisionControl;TextEditor;GNOME;GTK;
 StartupNotify=true
 Icon=ind.ie.Gnomit
+NoDisplay=true
```
?

sonnyp replied:
see https://github.com/indie-mirror/gnomit/pull/6

sonnyp commented:
@aral are you interested in contributions?

TingPing commented:
Might have better luck submitting it to the correct repo: https://source.ind.ie/gnome/gnomit/gjs

sonnyp commented:
I would but the GitLab instance doesn't allow registering.

aral commented:
Hey @sonnyp, thank you for the pull request. Sorry for the slow response; neck deep in other work at the moment. Will take a look when I get a moment.

sonnyp committed:
meson minimum version is 0.50

Fixes "WARNING: Project targeting '>= 0.40.0' but tried to use feature introduced in '0.50.0': install arg in configure_file"

sonnyp committed:
use stderr for errors

https://gitlab.gnome.org/GNOME/gjs/-/blob/master/doc/Logging.m
d#print-printerr

aral commented:
@sonnyp Finally managed to review. All looks good, thank you. Agree with hiding the desktop entry and the other changes. Appreciate your help :)
